test/unittest/tests/block/test_extent_list_insert_block.cpp

```diff
@@ -227,33 +227,26 @@ TEST_CASE("Extent Lists: block_append", "[extent_list]")
     {
         BREAK;
         /* Tests and expected values. */
-        std::vector<off_size_expected> test_list
-        {
-            {off_size(4096, 2048), // First half of First [4,096, 6,143].
-              {
-                off_size(4096, 2048), // First [4,096, 6,143].
-              }},
-              {off_size(4096 + 2048, 2048), // Adjacent: Second half of First [6,144, 8,191].
-                {
-                  off_size(4096, 4096), // First [4,096, 8,191].
-                }},
-#if 0 // FAILED: REQUIRE( last == extlist.last ) with expansion: 0x0000000040319c10 ==.
-      // 0x0000000040334d60. When FIXME-WT-13456 is fixed, enable this test. It is a reproducer.
+        std::vector<off_size_expected> test_list{
+          {off_size(4096, 2048), // First half of First [4,096, 6,143].
+            {
+              off_size(4096, 2048), // First [4,096, 6,143].
+            }},
+          {off_size(4096 + 2048, 2048), // Adjacent: Second half of First [6,144, 8,191].
+            {
+              off_size(4096, 4096), // First [4,096, 8,191].
+            }},
           {off_size(3 * 4096, 4096), // Not adjacent: Second [12,288, 16,383].
             {
               off_size(4096, 4096),     // First [4,096, 8,191].
               off_size(3 * 4096, 4096), // Second [12,288, 16,383].
             }},
```

Comment:
Do we have a test that causes the assert to fire?

Reply:
Now we do.

Reply:
Do they crash?

Reply:
Yes.